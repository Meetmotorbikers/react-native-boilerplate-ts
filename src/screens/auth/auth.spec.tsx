import { shallow } from 'enzyme';
import * as React from 'react';
import AuthScreen from './auth.screen';

describe('AuthScreen', () => {
  const wrapper = shallow(<AuthScreen componentId={'AuthScreen'} />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('contains a clickable button', () => {
    const button = wrapper.exists(`Button[testID="loginBtn"]`);

    expect(button).toBe(true);
  });
});
