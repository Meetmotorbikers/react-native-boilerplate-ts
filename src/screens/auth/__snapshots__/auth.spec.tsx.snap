// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthScreen renders correctly 1`] = `
<authscreen__Wrapper>
  <authscreen__Heading>
    Auth Screen
  </authscreen__Heading>
  <Button
    onPress={[Function]}
    testID="loginBtn"
    title="Login"
  />
</authscreen__Wrapper>
`;
